plugins/org.wso2.developerstudio.eclipse.gmf.esb.persistence/src/org/wso2/developerstudio/eclipse/gmf/esb/internal/persistence/LocalEntryTransformer.java

```diff
@@ -101,4 +101,19 @@ public Entry createEntry(FormPage visualLocalEntry) throws Exception {
 		return localEntry;
 	}
 
+	/**
+	 * returns the trimmed value
+	 * 
+	 * @param localEntryFormPage
+	 *            form page instance
+	 * @return trimmed value
+	 */
+	private String getTrimmedValue(LocalEntryFormPage localEntryFormPage) {
+		String localEntryFormPageValue = "";
```

Comment:
we don't need this variable we can do as follows:
if (localEntryFormPage.getLocalEntryTextValue() != null) {
    return localEntryFormPage.getLocalEntryTextValue().getText().replaceAll("[\\t\\n]", "");
}
return "";